examples/user_guide/03-Applying_Customization.ipynb

```diff
@@ -190,13 +189,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "To customize this visualization, you need to use option builder to associate your keywords with either the `Curve` or the `Spikes` object. To illustrate, note that in the first example, both the `Curve` and the `Spikes` have different `height` values set.\n",
+    "To customize this layout, you need to use an option builder to associate your keywords with either the `Curve` or the `Spikes` object, or else you would have had to apply the options to the individual elements before you built the composite object. To illustrate setting by type, note that in the first example, both the `Curve` and the `Spikes` have different `height` values provided.\n",
```

Comment:
Customizing the objects before composing them in the layout is not recommended - which is why I didn't mention it here.

Reply:
I think that sentiment is adequately conveyed in the rest of the document, and here it seemed important to be accurate to avoid confusion.